docker-compose.yml

```
@@ -1,5 +1,24 @@
 version: '3.8'
 
+x-app: &app
+  build:
+    context: .
+    target: hyku-base
+    args:
+      - EXTRA_APK_PACKAGES=less vim bash # TODO: this doesn't install vim, why?
+  image: registry.gitlab.com/notch8/britishlibrary:${TAG:-latest}
```

I'm guessing this should still be old image:
```suggestion
  image: ghcr.io/samvera/hyku:latest
```